### PR TITLE
prepare for v0.1.0 release

### PR DESCRIPTION
**Pull request by lauralt**
The description field is [a requirement](https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field) for publishing to crates.io.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v0.1.0
+
+This is the first `vm-device` release.
+The `vm-device` crate provides device traits defining read and write operations
+on specialized buses, a device manager for  operating devices and dispatching
```
Comment (gsserge) [inline CHANGELOG.md:7]: There's an extra space between "for" and "operating".

Comment (lauralt) [reply CHANGELOG.md]: fixed!

File: Cargo.toml [hunk 2]
```diff
@@ -2,6 +2,8 @@
 name = "vm-device"
 version = "0.1.0"
 authors = ["Samuel Ortiz <sameo@linux.intel.com>", "Liu Jiang <gerry@linux.alibaba.com>", "rust-vmm AWS maintainers <rust-vmm-maintainers@amazon.com>"]
+description = "management for virtual devices and resources"
+keywords = ["bus", "manager"]
```
Comment (gsserge) [inline Cargo.toml:6]: Maybe add "virtualization", "mmio" and "pio" to the list of keywords?

**Review by gsserge — CHANGES_REQUESTED** [review]

**Review by gsserge — APPROVED** [review]

**Comment by jiangliu** [comment]
Thanks for preparing this, I need some time to review current code, should be done early next week:)

**Comment by lauralt** [comment]
> Thanks for preparing this, I need some time to review current code, should be done early next week:)

Sure :+1:, early next week sounds good.

**Review by jiangliu — APPROVED** [review]

**Pull request merged**